owrrm/src/main/java/com/facebook/openwifi/rrm/RRMAlgorithm.java

```diff
@@ -35,6 +36,9 @@ public class RRMAlgorithm {
 	private static final Logger logger =
 		LoggerFactory.getLogger(RRMAlgorithm.class);
 
+	private static final ClientSteeringState clientSteeringState =
+		new ClientSteeringState();
```

Comment:
This is fine - can you add a comment that this is important and a only single instance needs to be passed to all algorithms? (This would have been obvious as a singleton, but otherwise you should add a note.)